boxsdk/object/file.py

```diff
@@ -44,20 +44,27 @@ def _get_accelerator_upload_url_for_update(self):
         """
         return self._get_accelerator_upload_url(file_id=self._object_id)
 
-    def content(self):
+    def content(self, bytes=None):
         """
         Get the content of a file on Box.
 
+        :peram bytes:
```

Comment:
Also, `bytes` is a built-in type in Python 3. Even though this is valid to do in Python, it still might be better to avoid that name. Something like `byte_range` or `byte_range_set` is more descriptive anyway.